### ISSUE TITLE
CSS problems on Firefox

### ISSUE THREAD

**Issue by SQL-MisterMagoo** [Blazored/Gitter#29]
Firefox is currently Scrollbar City Arizona - need to fix that.

![image](https://user-images.githubusercontent.com/11338430/56931177-125b5880-6ad7-11e9-8fd6-abaafae50fb9.png)

**Comment by chrissainty** [comment]
I'll have a look at what can be done here. The current scrollbar styling is only effective on webkit based browsers. Styling scrollbars seems poorly supported unfortunately.

**Comment by SQL-MisterMagoo** [comment]
The main problem is on Firefox the main app has scrollbars for some reason.
If can at least fix that the styling is what it is on Firefox, I really don't want to do slider scrollbars

**Comment by chrissainty** [comment]
Yes, I completely agree side scrolling is horrible. I think this is down to some padding or margin maths, I'll work it.
